src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="img-link"
+        href="./index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Company Logo"
+        />
+      </a>
+
+      <nav class="nav-container">
+        <ul class="nav-list">
+          <li class="nav-item highlighted">
+            <a
+              href="./"
+              class="nav-link is-active"
+            >
```

Comment:
The `href` attribute of the `a` tag is set to `./`, which is not a valid anchor link. It should start with the `#` symbol if it's meant to be an anchor link. If you want to link to the top of the page, you can just use `href="#"`. If you want to link to a different section of the page, you should use `href="#section-id"`, where `section-id` is the id of the section you want to link to.